### DISCUSSION TITLE
Web app and connections pool

### DISCUSSION THREAD

**Discussion by m-barthelemy** [go-rel/rel#333] [Q&A] [answered]
Hi,

If I understand correctly, creating an adapter with, for example, `postgres.Open(...)`, opens 1 connection under the hood.

Does `go-rel` have any support for pooling, especially with Postgres?
I'm using it in a web application that will be used by multiple users concurrently. How do I get `go-rel` to use a _pool_ of Postgres connections?

**Comment by Fs02** [comment, marked as answer]
REL is using sql.DB under the hood
you can check sql.DB for pool related config
https://pkg.go.dev/database/sql#DB.SetMaxIdleConns
https://go.dev/doc/database/manage-connections
